tutorial/accesstype-integration.md

```diff
@@ -0,0 +1,610 @@
+
+import "./subscription-page.m.css";
+
+const SubscriptionLayout = ({
+  initAccessType,
+  initRazorPayPayment,
+  getSubscription,
+  validateCoupon,
+  member,
+}) => {
+  const [activeTab, setActiveTab] = useState("subscription");
+  const [selectedPlan, setSelectedPlan] = useState({});
+
+  useEffect(() => {
+    !global.AccessType && initAccessType();
+  }, []);
+
+  switch (activeTab) {
+    case "subscription":
+      return (
+        <GroupsAndPlansModal
+          member={member}
+          setActiveTab={setActiveTab}
+          selectedPlan={selectedPlan}
+          setSelectedPlan={setSelectedPlan}
+          getSubscription={getSubscription}
+        />
+      );
+
+    case "login":
+      return (
+        <LoginModal
+          setActiveTab={setActiveTab}
+          AccountModal={AccountModal}
+          selectedPlan={selectedPlan}
+        />
+      );
+
+    case "checkout":
+      return (
+        <CheckoutModal
+          member={member}
+          setActiveTab={setActiveTab}
+          initRazorPayPayment={initRazorPayPayment}
+          selectedPlan={selectedPlan}
+          validateCoupon={validateCoupon}
+        />
+      );
+
+    default:
+      return (
+        <GroupsAndPlansModal
+          member={member}
+          setActiveTab={setActiveTab}
+          getSubscription={getSubscription}
+        />
+      );
+  }
+};
+
+SubscriptionLayout.propTypes = {
+  initAccessType: func,
+  initRazorPayPayment: func,
+  getSubscription: func,
+  validateCoupon: func,
+  member: object,
+};
+
+export const SubscriptionPage = () => {
+  const member = useSelector((state) => get(state, ["member"], null));
+  const email = get(member, ["email"], "");
+  const phone = get(member, ["metadata", "phone-number"], "");
+  const { key, accessTypeBkIntegrationId } = useSelector((state) =>
+    get(state, ["qt", "config", "publisher-attributes", "accesstypeConfig"], {})
+  );
+
+  return (
+    <>
+      <AccessType
+        enableAccesstype={true}
+        isStaging={true}
+        accessTypeKey={key}
+        email={email}
+        phone={phone}
+        accessTypeBkIntegrationId={accessTypeBkIntegrationId}
+      >
+        {({
+          initAccessType,
+          initRazorPayPayment,
+          getSubscription,
+          validateCoupon,
+        }) => (
+          <SubscriptionLayout
+            initAccessType={initAccessType}
+            initRazorPayPayment={initRazorPayPayment}
+            getSubscription={getSubscription}
+            validateCoupon={validateCoupon}
+            member={member}
+          />
+        )}
+      </AccessType>
+    </>
+  );
+};
+```
+
+- Let’s get into the details of each of the modals created in the subscriptionLayout component.
+
+### Login Modal:
+
+- This modal is shown to anonymous users. In this modal we will show the account modal along with the plan selected
+
+```jsx title="app/isomorphic/components/pages/subscription-page/Modals/LoginModal/index.js"
```

Comment:
did you check how this above line is coming in gh-pages local ?

Reply:
Yeah I checked it is not showing the file path. So I will remove this